### ISSUE TITLE
m.weight.data.normal_(0, math.sqrt(2. / n)) what does it mean?

### ISSUE THREAD

**Issue by zhang405744522** [JiaRenChang/PSMNet#106]

**Comment by JiaRenChang** [comment]
@zhang405744522 
It is weight initialization.
Please refer to 
[Kaiming He 2015 paper](https://www.cv-foundation.org/openaccess/content_iccv_2015/papers/He_Delving_Deep_into_ICCV_2015_paper.pdf)

**Comment by niraj2477** [comment]
Can anyone please tell me , how can I initialize  it in keras

**Comment by zgh1349786296** [comment]
for m in self.modules():
            if isinstance(m, nn.Conv2d):
                n = m.kernel_size[0] * m.kernel_size[1] * m.out_channels
                m.weight.data.normal_(0, math.sqrt(2. / n))
            elif isinstance(m, nn.BatchNorm2d):
                m.weight.data.fill_(1)
                m.bias.data.zero_()
